common/msgs/autoware_planning_msgs/msg/VelocityLimit.msg

```diff
@@ -1,2 +1,2 @@
 builtin_interfaces/Time stamp
-float64 data
+float32 max_velocity
```

Comment:
I guess you should modify motion_velocity_optimizer and vehicle_rviz_plugin as well.